api/PaymentRequest.json

```diff
@@ -4,14 +4,25 @@
       "__compat": {
         "mdn_url": "https://developer.mozilla.org/docs/Web/API/PaymentRequest",
         "support": {
-          "webview_android": {
-            "version_added": false
-          },
           "chrome": {
-            "version_added": "61"
+            "version_added": "61",
+            "flags": [
+              {
+                "type": "preference",
+                "name": "Service Worker payment apps",
+                "value_to_set": "Enabled"
+              }
+            ]
           },
           "chrome_android": {
-            "version_added": "53"
+            "version_added": "53",
+            "flags": [
+              {
+                "type": "preference",
+                "name": "Service Worker payment apps",
+                "value_to_set": "Enabled"
+              }
+            ]
           },
           "edge": {
             "version_added": true
```

Comment:
Microsoft Edge [supports this since version 15](https://blogs.windows.com/msedgedev/2016/12/15/payment-request-api-edge/).